integreat_cms/cms/models/contact/contact.py

```diff
@@ -52,6 +52,16 @@ class Contact(AbstractBaseModel):
         blank=True,
         verbose_name=_("phone number"),
     )
+    mobile_phone_number = models.CharField(
+        max_length=250,
```

Comment:
```suggestion
        max_length=40,
```

Just noticed. I would also chnge it for the other two. From a brief search online, it seems phone longers (worldwide) are never longer than 15 numbers. Let's be generous and add 10 characters for spaces, braces, and +, and another 15 for internal extensions, and then this is probably still overkill 😄